frontend/packages/shared-components/src/Svg.stories.js

```diff
@@ -0,0 +1,50 @@
+import { Svg } from './';
+
+export default {
+  title: 'Shared Components/Svg',
+  component: Svg,
+  argTypes: {
+    name: {
+      control: {
+        type: 'select',
+      },
+      options: [
+        'Active',
+        'AngleDown',
+        'AngleUp',
+        'ArrowLeft',
+        'ArrowLeftBold',
+        'ArrowRight',
+        'Bin',
+        'Calendar',
+        'CaretDown',
+        'CheckCircle',
+        'CheckMark',
+        'Close',
+        'Copy',
+        'Discord',
+        'Eye',
+        'GitHub',
+        'HideEye',
+        'Image',
+        'InfoOutLine',
+        'Instagram',
+        'InvalidCheckMark',
+        'LinkOut',
+        'Logo',
+        'Plus',
+        'Star',
+        'Twitter',
+        'Upload',
+        'ValidCheckMark',
+        'Website',
```

Comment:
unfortunately have to maintain a list of options for the storybook as there wasnt a pretty way to dynamically get all the filenames of the Svgs the way we do with dynamic `import`

Reply:
I wonder if there's a way to show all icons in one component, so we can easily view them/search for them